src/index.js

```diff
@@ -178,6 +176,7 @@ class Dropzone extends React.Component {
   }
 
   open() {
+    this.setState({ isFileDialogActive: true });
```

Comment:
Yes, that makes more sense since we don't actually want to re-render when changing the value. Just to track it.